synapse/handlers/oidc.py

```diff
@@ -301,11 +301,11 @@ def __init__(
                 hs.get_clock(),
             )
 
-        self._client_auth = ClientAuth(
+        self._client_auth: ClientAuth = ClientAuth(
```

Comment:
I think we can just remove this type hint?

Reply:
Oh wow, yeah, i'll remove this. :+1: